examples/generative/corrdiff/conf/sampler/stochastic.yaml

```diff
@@ -16,4 +16,5 @@
 
 type: stochastic
 boundary_pix: 2
-overlap_pix: 0
+overlap_pix: 4
```

Comment:
Do we know how this will affect the CWA model?

Reply:
`overlap_pix: 4` is supposed to be the default setting for all patch-based models. `overlap_pix: 0` will make the sampler output NANs